src/components/footer.js

```diff
@@ -1,17 +1,25 @@
 import React from 'react';
-import './footer.css';
+import styled from 'styled-components';
```

Comment:
@jlong5795 @mtruj013 @npm-nic I am nervous merging this branch KP's changes may result in that same merge conflict we struggled with before. I think KP may have to delete her local Front End repo and clone it again ? Or else this Footer file might give us issues again.